doc/build/faq/connections.rst

```diff
@@ -281,7 +281,7 @@ statement executions::
                               trans.rollback()
 
                       time.sleep(retry_interval)
-                      context.cursor = cursor_obj = connection.connection.cursor()
```

Comment:
please change the local variable name to "cursor_obj" in the _run_with_retries() call, thanks!

Reply:
reverted